start/node/src/main/java/org/ethereum/beacon/node/Node.java

```diff
@@ -109,6 +109,44 @@
   )
   private String metricsEndpoint;
 
+  @CommandLine.Option(
+      names = "--initial-state",
+      paramLabel = "initial-state",
+      description = {
+          "Path to an initial state file (SSZ format)"
+      }
+  )
+  private String initialStateFile;
+
+  @CommandLine.Option(
+      names = "--start-mode",
+      paramLabel = "start-mode",
+      description = {
+          "Specifies how to deal with the existing or absent storage. Possible modes:",
+          "  initial - starts from an empty storage only. If it's not, --force-db-clean can be specified.",
+          "  storage - starts from a previously initialized storage only,",
+          "            ignoring contract/initial-state parameters",
+          "  auto    - starts from an existing storage, if it's non empty",
+          "            initializes from contract/initial-state parameters otherwise",
+          "By default, set to auto, if no initial-state is specified,",
+          "            and to initial, if an initial-state is specified."
+      }
+  )
+  private String startMode;
+
+  @CommandLine.Option(
+      names = "--force-db-clean",
+      paramLabel = "force-db-clean",
+      description = {
+          "When an initial-state is specified, but db is not empty",
+          "specifies how to resolve the problem:",
+          "  force-db-clean=true - tries to clean db",
+          "  force-db-clean=true - exits with failure status.",
```

Comment:
There should be `false` here, shouldn't it?

Reply:
Definitely!